### ISSUE TITLE
Formatting not beeing applied

### ISSUE THREAD

**Issue by helisoncavalheiro** [open-southeners/vscode-laravel-pint#37]
**Describe the bug**
I've installed the extension in my VSCode but the Laravel Pint is not beeing ran on project nor files, as the GIF below shows.

**To Reproduce**
1. Install the extension.
2. Go to Settings extension
3. Check the "**Enable**" option
4. Uncheck the "**Fallback to Global Bin**"
5. Select "**Laravel" on "Preset**" option
6. Check "**Run In Laravel Sail**" option
7. Go to a PHP file
8. Unformat a line (e.g Adding extra spaces before a line)
9. Press **Ctrl + Shift + P**
10. Select "**Format document using Laravel Pint**"

**Expected behavior**
Laravel Pint beeing executed through Laravel Sail and the file beeing properly formatted.

**Screenshots**
![2023-04-26_16-53-11](https://user-images.githubusercontent.com/31669394/234689349-ca41ac85-bfa6-4847-9f3c-5aeae3febb2a.gif)

**Environment:**
 - Operating system: Fedora 36 - KDE Plasma
 - IDE / version: VSCode 1.77.3 - Electron/19.1.11 Safari/537.36
 - Extension version: 1.1.4
 - Extension's config:
 
```
{
     "laravel-pint.enable": true,
    "laravel-pint.runInLaravelSail": true,
    "laravel-pint.fallbackToGlobalBin": false,
    "laravel-pint.preset": "laravel",
}
```
**Additional context**
I've tested Laravel Pint through command line and it works as expected running `vendor/bin/sail php vendor/bin/pint`

**Comment by helisoncavalheiro** [comment]
UPDATE: Using [Dev containers](https://code.visualstudio.com/docs/devcontainers/containers) solves the problem.
However I'm still unable to run it outside the container.

**Comment by TheoMoriceEM** [comment]
Same problem here, works through command line but auto-format on save just doesn't no matter what I try, and despite the fact that the output says that the file has been formatted. I'm running my project in WSL.

**Comment by asivaneswaran** [comment]
Same here

**Comment by d8vjork** [comment]
If what @medialoungeml says is right this might get fixed by #51 (issue #50)

**Comment by d8vjork** [comment]
v1.1.6 published, will close this issue as it might get solved with this

Feel free to comment or reopen this if find something similar
